src/storage/file_manager.c

```diff
@@ -4247,7 +4261,7 @@ file_rv_perm_expand_undo (THREAD_ENTRY * thread_p, LOG_RCV * rcv)
   DKNSECTS save_nsects;
 
   fhead = (FILE_HEADER *) page_fhead;
-  file_header_sanity_check (fhead);
+  file_header_sanity_check (thread_p, fhead);
```

Comment:
I think sanity check must be removed from file_rv_perm_expand_undo and file_rv_perm_expand_redo. Expand is part of page allocation (or even page deallocation can get to do expand), and the file header may not be sane (some changes are not fully reflected). Sanity checks should only be done at the end of entry/exit points of file main operations: create/destroy, allocate/deallocate.